bench/GithubApiBenchmark.cs

```diff
@@ -158,14 +235,67 @@ public void Setup()
 
 public sealed record MergeBranchResponse([property: JsonPropertyName("url")] Uri Url,
                                          [property: JsonPropertyName("sha")] string Sha,
-                                         [property: JsonPropertyName("commit")] Commit Commit);
+                                         [property: JsonPropertyName("node_id")] string NodeId,
+                                         [property: JsonPropertyName("html_url")] Uri HtmlUrl,
+                                         [property: JsonPropertyName("comments_url")] Uri CommentsUrl,
+                                         [property: JsonPropertyName("commit")] Commit Commit,
+                                         [property: JsonPropertyName("author")] Author Author,
+                                         [property: JsonPropertyName("committer")] Author Committer,
+                                         [property: JsonPropertyName("parents")] ImmutableArray<Tree> Parents,
+                                         [property: JsonPropertyName("stats")] Stats Stats,
+                                         [property: JsonPropertyName("files")] ImmutableArray<File> Files);
 
 public sealed record Commit([property: JsonPropertyName("url")] Uri Url,
-                            [property: JsonPropertyName("author")] Author Author,
-                            [property: JsonPropertyName("committer")] Author Committer,
+                            [property: JsonPropertyName("author")] CommitAuthor Author,
+                            [property: JsonPropertyName("committer")] CommitAuthor Committer,
                             [property: JsonPropertyName("message")] string Message,
-                            [property: JsonPropertyName("comment_count")] int CommentCount);
+                            [property: JsonPropertyName("tree")] Tree Tree,
+                            [property: JsonPropertyName("comment_count")] int CommentCount,
+                            [property: JsonPropertyName("verification")] Verification Verification);
+
+public sealed record CommitAuthor([property: JsonPropertyName("name")] string Name,
+                                  [property: JsonPropertyName("email")] string Email,
+                                  [property: JsonPropertyName("date")] DateTime Date);
+
+public sealed record Tree([property: JsonPropertyName("url")] Uri Url,
+                          [property: JsonPropertyName("sha")] string Sha);
+
+public sealed record Verification([property: JsonPropertyName("verified")] bool Verified,
+                                  [property: JsonPropertyName("reason")] string Reason,
+                                  [property: JsonPropertyName("signature")] string? Signature,
+                                  [property: JsonPropertyName("payload")] string? Payload);
+
+public sealed record Author([property: JsonPropertyName("login")] string Login,
```

Comment:
Rename to `GitHubUser`.

Reply:
Done with hd191c43b32f5ac4a4942aa746bc6d28b25e516e2